llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -1411,11 +1411,19 @@ bool VPlanTransforms::tryAddExplicitVectorLength(VPlan &Plan) {
   // The transform updates all users of inductions to work based on EVL, instead
   // of the VF directly. At the moment, widened inductions cannot be updated, so
   // bail out if the plan contains any.
-  if (any_of(Header->phis(), [](VPRecipeBase &Phi) {
-        return (isa<VPWidenIntOrFpInductionRecipe>(&Phi) ||
-                isa<VPWidenPointerInductionRecipe>(&Phi));
-      }))
+  bool IncludeWidenInduction = any_of(Header->phis(), [](VPRecipeBase &Phi) {
```

Comment:
nit: `ContainsWideInductions` would be more accurate? Move comment to actual bail out?

Reply:
26fbee2e0af165f4cdcfcd7b4a69f5867484f554
I think the comment should stay where it is. This way, it can be more clearly understood which boolean variable it is referring to.